legend-engine-executionPlan-execution-store-relational-connection-tests-api/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/tests/api/connectionTestManagers/SqlServerConnectionTestManager.java

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.plan.execution.stores.relational.connection.tests.api.connectionTestManagers;
+
+import org.finos.legend.engine.plan.execution.stores.relational.connection.tests.api.ConnectionTestManager;
+import org.finos.legend.engine.plan.execution.stores.relational.connection.tests.api.DynamicTestConnection;
+import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.relational.connection.DatabaseType;
+import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.relational.connection.RelationalDatabaseConnection;
+import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.relational.connection.authentication.UserNamePasswordAuthenticationStrategy;
+import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.relational.connection.specification.StaticDatasourceSpecification;
+import org.finos.legend.engine.shared.core.vault.PropertiesVaultImplementation;
+import org.finos.legend.engine.shared.core.vault.Vault;
+import org.finos.legend.engine.shared.core.vault.VaultImplementation;
+import org.testcontainers.containers.MSSQLServerContainer;
+
+import java.util.Properties;
+
+public class SqlServerConnectionTestManager implements ConnectionTestManager
+{
+
+    @Override
+    public DatabaseType getDatabaseType()
+    {
+        return DatabaseType.SqlServer;
+    }
+
+    @Override
+    public String getPureTestCollectionPath()
```

Comment:
Note to self. Take a closer look

Reply:
Probably we should rename it to DatabaseTestManager. Providing a on the fly tests connection is just one part of it. Mostly it is to specify the reference to db specific tests. (and optionally a test connection to use to run them)

Reply:
defined the test paths directly now